README.md

```diff
@@ -127,6 +127,7 @@ This mod is provided free of charge, but I do accept donations. If you'd like to
 * nhydock
 * RPKU
 * Sauceke
+* hogefugamoga
```

Comment:
This file gets overwritten after each release, please edit the doc/README.md file instead.